telegram/ext/regexhandler.py

```diff
@@ -72,9 +97,11 @@ def __init__(self,
                  pass_job_queue=False,
                  pass_user_data=False,
                  pass_chat_data=False,
-                 allow_edited=False,
                  message_updates=True,
```

Comment:
I made the same modifiaction in MessageHandler (in an earlier pr) and it went through without any trouble.